mapper/orm-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/outboxpolling/cfg/impl/UuidDataTypeUtils.java

```diff
@@ -61,12 +61,12 @@ else if ( UUID_BINARY.equalsIgnoreCase( value ) ) {
 		return TypeCodeConverter.convert( value );
 	}
 
-	public static int defaultUuidType(Dialect dialect) {
+	public static Integer defaultUuidType(Dialect dialect) {
 		// See HSEARCH-4749 why MSSQL is treated differently
 		if ( dialect instanceof SQLServerDialect ) {
 			return SqlTypes.BINARY;
 		}
```

Comment:
Do we still need this `if` block? Binary would be the default on that dialect anyway, no?

Reply:
let me double-check if that's the default for MSSQL as well.. also I left it in as this was something that we wanted to force (even if something changes in the dialect we'd still pick the binary type ourselves)